### PR TITLE
Fix that weird overlapping CSS bug in Empire plots

### LINKED ISSUE #356: Emperor isn't perfectly aligned in empire plots
Not a big deal, but now the "31 / 31 visible" text is partially overlapped by the Empress visualization:

image

This is present in the master branch of Empress. Not sure where this came from -- probably some minor CSS that's messing with this.

### PR DESCRIPTION
**Pull request by fedarko**
Closes #356.

Also updated the screenshots/GIFs of Empire plots in the README, since this visual error was present in these as well. And added a small footnote to part of the README giving a caveat about the ignore absent tips setting, since I realized this could be confusing in some cases.

### CODE REVIEW

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empire-biplot.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F416%2Fempire-biplot.qzv), [empire.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F416%2Fempire.qzv), [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F416%2Fempress-tree.qzv), [just-fm.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F416%2Fjust-fm.qzv), [plain.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F416%2Fplain.qzv)

File: README.md [hunk 1]
```diff
@@ -374,12 +374,14 @@ You may have noticed that in the Emperor plot one of the *Right Palm* samples is
 
 ![empire_sample_selection_gut_and_palm](docs/moving-pictures/img/empire_sample_selection_gut_and_palm.gif)
 
-Once the samples have been deselected (within a couple of seconds), select the outlier palm sample + one of the gut samples. What do you notice? You’ll see that comparatively few unique red or orange branches light up, suggesting that this sample shares many more features with the gut samples than the other palm samples.
+Once the samples have been deselected (within a couple of seconds), select the outlier palm sample + one of the gut samples. What do you notice? You’ll see that comparatively few unique red or orange branches light up, suggesting that this sample shares many more features with the gut samples than the other palm samples.\*
```
Comment (ElDeveloper) [inline README.md:377]: What's the `\*` for?

Comment (fedarko) [reply README.md:377]: It's pointing to the new paragraph on the ignore absent tips setting. The `\*` prevents markdown from interpreting the `*` as the start of italics.

It makes this part of the README look like this in practice:

int

Comment (ElDeveloper) [reply README.md:377]: Didn't know that, thanks!

**Comment by ElDeveloper** [comment]
Thanks @fedarko!

**Pull request merged**